src/arrays/multidimensional_arrays/hard/q3/CountsOdd.java

```diff
@@ -1,25 +1,41 @@
 package arrays.multidimensional_arrays.hard.q3;
 
 /*
-Make a Java program that defines a 3 by 3 multidimensional array in the main method and calls the countsOdd method. The
-countsOdd method should accept a multidimensional array of integers of any size, count the number of odd integers in the
-array, and print the count.
+Write a Java program that randomly initialize a 3 by 3 multidimensional array, with integers between 1 and 100
+(inclusive of both ends) and prints it. Then gets two integers a and b and count the number of odd number for all
+the integers in the array whose indices are i and j so that 0 <= i < a and 0 <= j < b.
```

Comment:
This question text needs to be clarified. I don't understand what the question is asking